src/ui/components/UploadScreen/UploadScreen.tsx

```diff
@@ -157,7 +157,7 @@ export default function UploadScreen({ recording, userSettings, onUpload }: Uplo
     e.preventDefault();
 
     setStatus("saving");
-    const workspaceId = selectedWorkspaceId == "" ? null : selectedWorkspaceId;
+    const workspaceId = selectedWorkspaceId == MY_LIBRARY ? null : selectedWorkspaceId;
```

Comment:
Can `selectedWorkspaceId` itself be changed to be `null` by default? Checks like this are super easy to break.